jest.u.setup.ts

```diff
@@ -0,0 +1 @@
+import '@testing-library/jest-dom'
```

Comment:
I don't think this is necessary (and by proxy don't think this setup file is necessary -- at least yet). We aren't doing any DOM testing. Note that our `testEnvironment` is `node` not `browser`

Also this package isn't in your `package.json` anyway...